iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/output/report/primary-processing-execution-times.json

```diff
@@ -13,5 +13,4 @@
 {"key": "processing.referenceExtraction.patent.duration", "type": "DURATION", "value": "$(LONG_RANGE: [1000, 700000])"}
 {"key": "processing.docOrgMatching.affiliationBased.duration", "type": "DURATION", "value": "$(LONG_RANGE: [1000, 200000])"}
```

Comment:
Let me add one comment here: I have found `processing.docOrgMatching.duration` entry in the spreadsheet but I don't think we have added this metric (as it wasn't present before).

Reply:
I think actually we're missing two reports here. We have three algos for docOrgMatching: affiliationBased, projectBased, deduped; only affiliationBased and projectBased are covered by reports. So I think we need one more report for deduped algo and one more for total docOrgMatching:
* `processing.docOrgMatching.deduped.duration`
* `processing.docOrgMatching.duration`
Am I right ?

Reply:
Well, I'd go with `processing.docOrgMatching.duration` but skip `processing.docOrgMatching.deduped.duration` as pretty irrelevant. We don't measure deduplication for other algos as well.